mkdocs.yml

```diff
@@ -1,23 +1,25 @@
 site_name: Documentation
 nav:
     - Home: index.md
-    - Developer Portal: http://testtool.dhq.onem:6060/
-    - Getting Started:
-        - Node.js: getting-started/getting-started-nodejs.md
-        - Python: getting-started/getting-started-python.md
     - Introduction:
         - What are ONEm apps?: intro.md
         - Core concepts: concepts.md
+        - High level overview: high-level-overview.md
         - Build your first app: buildfirstapp.md
+    - Getting Started:
+        - Node.js: getting-started/getting-started-nodejs.md
+        - Python: getting-started/getting-started-python.md
     - Building:
         - Webhooks: building/webhooks.md
```

Comment:
The building/webhooks section states that the dev is going to define `user_action` webhooks when registering the app on the dev portal. I am assuming this will be added in the `Applications > Create an app` form on the portal ? (along with app name, descr, website url, callback url)

Reply:
Its probably a term for the things (user requests to onem platform, onem platform requests to 3rd party) which are already happening. Dont know if the dev needs to add them since its a default. @chrishornmem could clear this up

Reply:
The documents are not quite in sync with the reality.  I had the idea of having multiple webhook event types that the developer could associate different URLs for each one.  For example, they could get notified when a blockchain event happens, or a mcoin transaction or a user to user message or something.  Another approach is for us to support only one general webhook URL per app, but have webhook event types.